src/screens/SettingsScreen.js

```diff
@@ -244,7 +241,7 @@ export const SettingsScreen = () => {
       )}
       {selectedFilterId === TOP_FILTER.LIST_TYPES && <ListSettings />}
       {selectedFilterId === TOP_FILTER.AR_DOWNLOAD_LIST && (
-        <AugmentedReality id="579" onSettingsScreen />
+        <AugmentedReality id={settings?.ar?.tourId} onSettingsScreen />
```

Comment:
if there is `!!settings.ar` this means that `settings.ar` is accessible, so you do not need the optional chaining here. please access `settings.ar.tourId` directly

Reply:
Yes! You're right! 👍
https://github.com/ikuseiGmbH/smart-village-app-app/pull/661/commits/639962767920160946ddf04cb47003983ba368e2

Reply:
or am i wrong? the check for `!!settings.ar` is not related to the rendering part here, right? but the selected filter is only visible, if `!!settings.ar` was true before, right? so we can be sure, that `settings.ar` is accessible here?